### DISCUSSION TITLE
挂载的阿里云盘open 差不多一天挂一次

### DISCUSSION THREAD

**Discussion by Markkkkks** [AlistGo/alist#4746] [Q&A]
挂载的阿里云盘open 差不多一天挂一次 需要重新获取token
alist版本：v3.19.0-aarch64
挂载失败截图
![image](https://github.com/alist-org/alist/assets/17808209/0f48b181-b8eb-4ebc-96e6-001c0c80752e)
重新挂载：
![image](https://github.com/alist-org/alist/assets/17808209/0381cc13-ff74-47ee-a79d-94931f4b56a5)
![image](https://github.com/alist-org/alist/assets/17808209/60c060c5-016f-4809-b18b-78938a360f9b)
![image](https://github.com/alist-org/alist/assets/17808209/b0e5249a-5c15-4cbc-9a91-0ca3c3ccf350)

**Comment by ZealousLyon** [comment]
> 同遇到该问题，CF的和辽宁的都提示TCP连接失败

2023-07-13 Update
CF的地址已恢复正常使用

**Comment by ksewin** [comment]
我的是：AccessTokenExpired: token expired
